test/DaedalusCompiler.Tests/CompilerFullTest.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Linq;
+using DaedalusCompiler.Compilation;
+using DaedalusCompiler.Dat;
+using Xunit;
+
+namespace DaedalusCompiler.Tests
+{
+    public class CompilerFullTest
+    {
+        private string _code;
+        private AssemblyBuilder _assemblyBuilder;
+
+        public CompilerFullTest()
+        {
+            _assemblyBuilder = new AssemblyBuilder();
+        }
+
+        private DatFile compileCodeAndGetDATFileOfThat()
```

Comment:
functions should follow single responsibility principle so word 'and' in function's name is not good, it should be changed to something else or function should be split.